src/main/scala/temple/test/internal/EndpointTest.scala

```diff
@@ -0,0 +1,18 @@
+package temple.test.internal
+
+private[internal] class EndpointTest(service: String, endpointName: String) {
+  class TestFailedException(msg: String) extends RuntimeException(msg)
+
+  def assertEqual(expected: Any, found: Any): Unit =
+    if (!expected.equals(found))
+      fail(s"Expected $expected but found $found")
+
+  def assert(value: Boolean, failMsg: String): Unit =
+    if (!value) fail(failMsg)
+
+  def pass(): Unit =
+    println(s"\t✅ $service $endpointName")
```

Comment:
`\t` will be inconsistently wide between terminals, does this look very weird if a tab is expanded to width eight?

Reply:
If only we had an indent function... 😉